ibis/expr/operations/generic.py

```diff
@@ -262,8 +262,6 @@ class Literal(Value):
                     decimal.Decimal,
                 )
             ),
-            # this seems buggy
-            rlz.is_computable_input,
```

Comment:
Actually, can we keep this line, and fully remove in 4.0.0?

Reply:
That would raise deprecation warnings I think resulting pytest errors.

Reply:
We can selectively ignore warnings to prevent that.